node/test/graphql/mutation/me/invalidate-responses.spec.ts

```diff
@@ -0,0 +1,382 @@
+/*
+This software is Copyright ©️ 2020 The University of Southern California. All Rights Reserved. 
+Permission to use, copy, modify, and distribute this software and its documentation for educational, research and non-profit purposes, without fee, and without a written agreement is hereby granted, provided that the above copyright notice and subject to the full license file found in the root of this software deliverable. Permission to make commercial use of this software may be obtained by contacting:  USC Stevens Center for Innovation University of Southern California 1150 S. Olive Street, Suite 2300, Los Angeles, CA 90115, USA Email: accounting@stevens.usc.edu
+
+The full terms of this copyright and license should always be found in the root directory of this software deliverable as "license.txt" and if these terms are not found with this software, please contact the USC Stevens Center for the full license.
+*/
+import createApp, { appStart, appStop } from 'app';
+import { expect } from 'chai';
+import { Express } from 'express';
+import { describe } from 'mocha';
+import mongoUnit from 'mongo-unit';
+import request from 'supertest';
+import { getToken } from '../../../helpers';
+
+describe('invalidateResponses', () => {
+  let app: Express;
+
+  beforeEach(async () => {
+    await mongoUnit.load(require('test/fixtures/mongodb/data-default.js'));
+    app = await createApp();
+    await appStart();
+  });
+
+  afterEach(async () => {
+    await appStop();
+    await mongoUnit.drop();
+  });
+
+  it(`throws an error if not logged in`, async () => {
+    const response = await request(app)
+      .post('/graphql')
+      .send({
+        query: `mutation InvalidateResponse($expectation: Int!, $invalid: Boolean!, $invalidateResponses: [InvalidateResponseInputType!]) {
```

Comment:
make this static gql query a top level const like `GQL_INVALIDATE_RESPONSE_DEFAULT` and then replace all the instances of it with the variable (seems like at least 4 or 5?)